lib/libzfs/libzfs.abi

```diff
@@ -234,6 +235,7 @@
     <elf-symbol name='membar_enter' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
     <elf-symbol name='membar_exit' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
     <elf-symbol name='membar_producer' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='membar_sync' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
```

Comment:
I guess we skipped this when membar_sync was added. It is a little strange to appear in this commit but I don't mind.

Reply:
I have moved this change to a new commit.